tests/storage/storage.go

```diff
@@ -137,23 +137,25 @@ var _ = SIGDescribe("Storage", func() {
 				}
 			}
 		}
-		Context("with error disk", func() {
+		Context("[Serial]with error disk", Serial, func() {
 			var (
 				nodeName, address, device string
 
 				pvc *k8sv1.PersistentVolumeClaim
+				pv  *k8sv1.PersistentVolume
 			)
 
 			BeforeEach(func() {
 				nodeName = tests.NodeNameWithHandler()
 				address, device = tests.CreateErrorDisk(nodeName)
 				var err error
-				_, pvc, err = tests.CreatePVandPVCwithFaultyDisk(nodeName, device, testsuite.GetTestNamespace(nil))
+				pv, pvc, err = tests.CreatePVandPVCwithFaultyDisk(nodeName, device, testsuite.GetTestNamespace(nil))
 				Expect(err).NotTo(HaveOccurred(), "Failed to create PV and PVC for faulty disk")
 			})
 
 			AfterEach(func() {
 				tests.RemoveSCSIDisk(nodeName, address)
+				Expect(virtClient.CoreV1().PersistentVolumes().Delete(context.Background(), pv.Name, metav1.DeleteOptions{})).NotTo(HaveOccurred())
```

Comment:
Would it make sense to clean up the PVC too?

Reply:
The PVC is namespaced, so it should automatically be cleaned up